src/libCrypto/EthCrypto.cpp

```diff
@@ -389,7 +389,7 @@ bytes GetOriginalHash(TransactionCoreInfo const& info, uint64_t chainId) {
   rlpStreamRecreated << info.nonce;
   rlpStreamRecreated << info.gasPrice;
   rlpStreamRecreated << info.gasLimit;
-  bytes toAddr{};
+  bytes toAddr(20, 0);  // Default is empty address
```

Comment:
For contract creation, the tx hash generation would not work because it didn't reconstruct the RLP correctly (it needs 40 0s here.

I do not think this is an issue, but we will get into trouble if leaving the address blank rather than a null address is also a valid way to create a contract.

Reply:
I think this is not correct.  I had so much debugging and confusion in the past, I don't think you should revert it.

<img width="464" alt="image" src="https://user-images.githubusercontent.com/114107/191525517-dbd5f892-bdc8-44e7-8e38-b9e4cb92ffd4.png">